gen3-client/g3cmd/auth.go

```diff
@@ -0,0 +1,58 @@
+package g3cmd
+
+import (
+	"fmt"
+	"log"
+	"sort"
+
+	"github.com/spf13/cobra"
+	"github.com/uc-cdis/gen3-client/gen3-client/jwt"
+)
+
+func init() {
+
+	var authCmd = &cobra.Command{
+		Use:     "auth",
+		Short:   "Return data access priveleges from profile",
+		Long:    `Gets data access priveleges for specified profile.`,
+		Example: `./gen3-client auth --profile=<profile-name>`,
+		Run: func(cmd *cobra.Command, args []string) {
+
+			request := new(jwt.Request)
+			configure := new(jwt.Configure)
+			function := new(jwt.Functions)
+
+			function.Request = request
+			function.Config = configure
+
+			endPointPostfix := "/user/user" // Information about current user
+
+			host, project_access, err := function.CheckPrivileges(profile, "", endPointPostfix, "application/json", nil)
+
+			if err != nil {
+				log.Fatalf("Fatal authentication error: %s\n", err)
+			} else {
+				if len(project_access) == 0 {
+					fmt.Printf("\nYou don't currently have access to data from any projects at %s\n", host)
```

Comment:
if an ouput message is very meaningful you could use `log.Printf()` or `log.Println()`. In gen3-client I have set to let  these `log.x()` methods also write to a local log file in order to keep a record

Reply:
Thanks! Changed in the commit 0878755.